Project/Project/Pages/Index.cshtml.cs

```diff
@@ -22,43 +22,46 @@ public IndexModel(ILogger<IndexModel> logger)
         }
 
         public bool SearchCompleted { get; set; }
-        public string Query { get; set; }
+        public string CountryQuery { get; set; } 
```

Comment:
Changed the name of the field of Query, which was not clear about what it was determining, to CountryQuery. This was done keeping the Coding conventions in mind.